splice_machine/src/test/java/com/splicemachine/derby/impl/sql/compile/StringTypesToNumericTypeConversion.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2012 - 2017 Splice Machine, Inc.
```

Comment:
2018
btw, this IT file does not end with IT.java like others, will unit test run pick this?

Reply:
I'm preparing PR for master and branch-2.7, I can add the suffix of IT to the file to be consistent.